azure_pipelines/pr-validation.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       testRunTitle: 'Test Run - $(target)'
 
 - job: e2e_test_native_auth
-  displayName: 'Run MSAL E2E tests for native auth'
+  displayName: 'Run MSAL E2E tests for iOS native auth'
```

Comment:
Did we decide to run mac E2E tests also during PR validation? I thought we wanted to run those tests nightly and before the release (only in automation)

Reply:
I have now removed them as they also fail in the other case

Reply:
Are those tests run in parallel with the iOS E2E tests?

Reply:
They would be run in parallel on a different machine indeed